app/uk/gov/hmrc/vatsubscription/services/VatKnownFactsRetrievalService.scala

```diff
@@ -60,10 +62,14 @@ object VatKnownFactsRetrievalService {
 
   sealed trait GetVatCustomerInformationFailure extends GetVatKnownFactsFailures
 
-  case object InvalidVatNumber extends GetVatCustomerInformationFailure
+  case object InvalidVatNumber extends GetVatKnownFactsFailures
```

Comment:
I think the `InvalidVatNumber` and `VatNumberNotFound` objects should extend `GetVatCustomerInformationFailure` like they did previously, as they are more general connector errors and not specific get known facts failures.